.github/workflows/publish-release-npm.yml

```diff
@@ -38,7 +38,9 @@ jobs:
       - name: Publish NPM packages
         run: |
           npm config set '//registry.npmjs.org/:_authToken' "${NPM_TOKEN}"
-          DIST_TAG=$(cat lerna.json | jq '.version' --raw-output | xargs npm exec -- @k8slens/semver --prerelease 0)
+          VERSION=$(cat lerna.json | jq '.version' --raw-output)
+          echo ${VERSION}
```

Comment:
did you mean to leave this? Personally, I think it is a good idea, helps with debugging if there are issues in the future

Reply:
Yes I did mean to leave this in.